src/app/visits/page.tsx

```
@@ -1,23 +1,66 @@
-        <article className="rounded-3xl border border-[#CCE6EB] bg-white p-5 shadow-sm">
-          <p className="text-xs font-semibold uppercase tracking-[0.18em] text-[#5B8784]">
-            Patient Visits
-          </p>
-          <h2 className="mt-2 text-lg font-semibold text-[#1B5360]">Visit Schedule</h2>
-          <p className="mt-1 text-sm text-[#54797C]">Track and record patient clinic visits and checkups.</p>
-        </article>
+      <section className="grid gap-6 md:grid-cols-2">
+        <div className="rounded-3xl border border-[#E8F6F5] bg-white p-6 shadow-sm">
+          <div className="flex items-start justify-between">
+            <div>
+              <p className="text-xs font-semibold uppercase tracking-wide text-[#5B8784]">Today's schedule</p>
```

ESLint

[error] `'` can be escaped with `&apos;`, `&lsquo;`, `&#39;`, `&rsquo;`.

(react/no-unescaped-entities)